packages/front-end/pages/getstarted.tsx

```diff
@@ -424,6 +425,9 @@ const GetStartedPage = (): React.ReactElement => {
             </div>
           </div>
           <div className="col-auto pl-0">
+            <Button className="my-3" onClick={() => router.push("/setup")}>
```

Comment:
Who is allowed to see this/enter the flow? At first I was thinking only admins but not sure if we want to also support engineers/data scientists that could come in to do the sdk and/or data source steps within the flow